build01/buildkite.nix

```
@@ -13,5 +13,14 @@
       pkgs.nix
       pkgs.gzip
     ];
+
+    hooks.environment = ''
+      #!/usr/bin/env bash
+      set -euo pipefail
+
+      if [[ "$BUILDKITE_PIPELINE_SLUG" == "nixpkgs-swh" ]]; then
```

you might want to limit access to the master branch. not sure if that's enough to prevent third-party users from stealing the token though.
```suggestion
      if [[ "$BUILDKITE_PIPELINE_SLUG" == "nixpkgs-swh" && "$BUILDKITE_BRANCH" == master ]]; then
```